compconfig/compconfig/functions.py

```diff
@@ -95,6 +95,7 @@ def comp_output(calc1, calc2, out_var='mettr'):
     plt = calc1.grouped_bar(calc2, output_variable=out_var)
     js, div = components(plt)
     comp_dict = {
+        "model_version": "1.0.0",
```

Comment:
You could do `import ccc` or `from ccc import __version__` and set the `model_version` to `ccc.__version__`.

Reply:
Good suggestion! Just added this.